cmd/completion/main.go

```diff
@@ -28,25 +36,23 @@ const (
 )
 
 var (
+	cacheTag                string
 	notaryV1URL             string
 	dockerCredentials       flaghelpers.CredentialsFlags
 	dockerCfgCredentials    flaghelpers.CredentialsFlags
 	dockerConfigCredentials flaghelpers.CredentialsFlags
 	cosignAnnotations       flaghelpers.CredentialsFlags
 	cosignRepositories      flaghelpers.CredentialsFlags
 	cosignDockerMediaTypes  flaghelpers.CredentialsFlags
-	basicGitCredentials     flaghelpers.CredentialsFlags
-	sshGitCredentials       flaghelpers.CredentialsFlags
 	logger                  *log.Logger
 )
 
 func init() {
+	flag.StringVar(&cacheTag, "cache-tag", os.Getenv(buildapi.CacheTagEnvVar), "Tag of image cache")
 	flag.StringVar(&notaryV1URL, "notary-v1-url", "", "Notary V1 server url")
 	flag.Var(&dockerCredentials, "basic-docker", "Basic authentication for docker of the form 'secretname=git.domain.com'")
 	flag.Var(&dockerCfgCredentials, "dockercfg", "Docker Cfg credentials in the form of the path to the credential")
 	flag.Var(&dockerConfigCredentials, "dockerconfig", "Docker Config JSON credentials in the form of the path to the credential")
```

Comment:
What is the difference between these two flags? Is a `Docker Cfg` file different from a `Docker Config Json` file?

Reply:
Yes. Two different types of docker secrets: https://kubernetes.io/docs/concepts/configuration/secret/#secret-types

Reply:
so confusing right!